docs/quickstart.rst

```diff
@@ -8,51 +8,140 @@ up pg_auto_failover to replicate data between them. We’ll simulate failure in
 the primary node and see how the system smoothly switches (fails over)
 to the secondary.
 
-For simplicity we’ll run all the pieces on a single machine, but in
-production there would be three independent machines involved for each
-managed PostgreSQL service. One machine for the primary, another for
-the secondary, and the last as a monitor which watches the other nodes’
-health, manages global state, and assigns nodes their roles.
+For illustration, we'll run our databases on virtual machines in the Azure
+platform, but the techniques here are relevant to any cloud provider or
+on-premise network. We'll use three virtual machines: a primary database, a
+secondary database, and a monitor which watches the other nodes’ health,
+manages global state, and assigns nodes their roles.
+
+.. _quickstart_network:
+
+Create virtual network
+----------------------
+
+Our database machines need to talk to each other and to the monitor node, so
+let's create a virtual network.
+
+.. code-block:: bash
+
+   az group create \
+       --name ha-demo \
+       --location eastus
```

Comment:
I've been desultory reading the output of `az vm list -g ha-demo --query '[0]'` and I discovered the notions of availabilitySet and proximityPlacementGroup. After reading the docs about them I think we should create a PPG for our deployment here, and then put the two Postgres VMs (primary, secondary) in a single availability set.

Reading https://docs.microsoft.com/en-gb/azure/virtual-machines/linux/proximity-placement-groups and https://docs.microsoft.com/en-us/cli/azure/vm/availability-set?view=azure-cli-latest#az-vm-availability-set-create it looks like we could do:

```
az ppg create \
   -n ha-demo-ppg \
   -g ha-demo \
   -l eastus \
   -t standard

az vm availability-set create -n ha-demo-aset -g ha-demo --platform-fault-domain-count 2 --platform-update-domain-count 2 --ppg ha-demo-ppg
```

Then we can use `--ppg ha-demo-ppg` in the `az vm create` command for the monitor, and `--availability-set ha-demo-aset --ppg ha-demo-ppg` when creating the next two VMs.

Reply:
Still need to do this, after we review the current functionality